src/routers/courseItem.ts

```diff
@@ -1,23 +1,32 @@
 import express from 'express';
+import { create, read, update, del } from '../operators/courseItemOperations';
 
 const cItemRouter = express.Router();
+const app = express();
+
+app.use(express.json());
```

Comment:
Note from testing on PAS-101 which followed this same code structure for semesters:

I think this overwrites app defined in index.ts. When testing sending requests I was getting undefined for req.body. Removing line 5 and moving line 7 to index.ts solved it (thanks Braiden for solving the issue!)

Reply:
Thank you!